techniques/systemSettings/process/category.xml

```diff
@@ -16,5 +16,5 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 <xml>
     <name>Process</name>
```

Comment:
I talked with @fanf about renaming categories, and it seems it is not properly doable for now.